aptos-move/framework/aptos-framework/sources/reconfiguration_with_dkg.move

```diff
@@ -15,7 +16,13 @@ module aptos_framework::reconfiguration_with_dkg {
     /// Trigger a reconfiguration with DKG.
     /// Do nothing if one is already in progress.
     public(friend) fun try_start() {
```

Comment:
for `try_start`, prover test failed with
```
error: verification out of resources/timeout (global timeout set to 40s)
```
but it looks obvious to me that `dkg::incomplete_session()` and the following new code won't abort.

@rahxephon89 anything i'm missing?

Reply:
The current spec is actually broken because some specs are missing when landed on main. I am working on a fix for it: https://github.com/aptos-labs/aptos-core/pull/12364